nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
+        pass
```

Comment:
Also you don't need parentheses around the if condition, we are not writing C.

```python
if bla and foo:
```

rather than:
```python
if (bla and foo):
```